### PR TITLE
[-] BO : AdminImportController -> adresses add DNI

### PR DESCRIPTION
**Pull request by indesign47**

### CODE REVIEW

**Comment by jeromenadaud** [comment]
Hi, Thank you for your PR.

Best regards.

File: controllers/admin/AdminImportController.php [hunk 1]
```diff
@@ -72,7 +72,7 @@ class AdminImportControllerCore extends AdminController
 		'available_now' => array('AdminImportController', 'createMultiLangField'),
 		'available_later' => array('AdminImportController', 'createMultiLangField'),
 		'category' => array('AdminImportController', 'split'),
-		'online_only' => array('AdminImportController', 'getBoolean')
+		'online_only' => array('AdminImportController', 'getBoolean'),
```
Comment (mauroagr) [inline controllers/admin/AdminImportController.php]: This is Right? Put a comma at the end?

Comment (gRoussac) [inline controllers/admin/AdminImportController.php]: Hi,

In php yes you can do it normally. Some think it helps the next commiter to add rows. Like holding the door in the sub.

Regards

**Comment by SebSept** [comment]
Plus, there will be no change in this line when someone will add content on the next line.
That´s good. (Just like Using braves for à contrôle structure, éventuellement if it´s one Line ;) ^^)

https://github.com/PrestaShop/PrestaShop-norm-validator/pull/8

**Comment by gRoussac** [comment]
:trollface:

**Pull request merged**